internal/models/model_constants.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+const (
+	ProviderTypeUnknown int = iota
+	ProviderTypeAWS
+	ProviderTypeAzure
+	ProviderTypeGCE
+)
```

Comment:
Instead of introducing a table with providers, I propose to use constants (1 = AWS, 2 = Azure, 3 = GCE).